### ISSUE TITLE
Задокументировать процесс заливки geojson

### ISSUE THREAD

**Issue by olgapshen** [NorthGateVologda/NorthGateWiki#13]
Задокументировать процесс заливки файлов `geoJSON` в базу с помощью `ogr2ogr`.

**Comment by olgapshen** [comment]
@sshiae Тём, ты же вроде задокументировал, нет?

**Comment by sshiae** [comment]
Да, задокументировал, вот ссылка: https://github.com/NorthGateVologda/NorthGateWiki/blob/main/POSTGRES.md

**Comment by olgapshen** [comment]
> Да, задокументировал, вот ссылка: https://github.com/NorthGateVologda/NorthGateWiki/blob/main/POSTGRES.md

Круто :) Закрывай :)

**Comment by olgapshen** [comment]
@sshiae Не забывай пожалуйста актуализировать задачи.

**Comment by olgapshen** [comment]
@sshiae Тём...
